internal/backuprestore/backup.go

```diff
@@ -423,6 +424,11 @@ func (h *BRHandler) ExportRestoresToDir(ctx context.Context, configMaps []lcav1a
 
 	restores, err := extractRestoreFromConfigmaps(ctx, h.Client, configmaps)
 	if err != nil {
+		// detects validation error with dryrun
+		if !k8serrors.IsNotFound(err) {
+			ErrInvalidCR = err
+			return ErrInvalidCR
+		}
```

Comment:
```suggestion
	if err != nil {
	         return err
         }
```
There are other error types other than notfound and invalid.  For particular invalid error type returned by the API call should be k8serrors.IsInvalid(err).  And in the main handlerUpgrade function, you should check the if it's k8serrors.IsInvalid(err)

Reply:
Updated to use Invalid